app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -19,5 +19,6 @@
 
 <dl class="form">
   <dt><label>Add your starter code from GitHub <small>(optional)</small> </label></dt>
-  <dd><input class="textfield" type="text" name="repo_name" value="<%= assignment.full_name %>" placeholder="<%= assignment.full_name || 'e.g. education/classroom'%>"</dd>
+  <dd><input class="textfield js-autocomplete-textfield" type="text" name="repo_name" value="<%= assignment.full_name %>" placeholder="<%= assignment.full_name || 'e.g. education/classroom'%>" autocomplete="off" data-autocomplete-search-endpoint="github_repos"></dd>
```

Comment:
Previously, we used the `name` attribute to determine the endpoint to send request to. 

As we changed the `routes.rb`, I then add a custom [`data-*`](https://www.w3.org/TR/html5/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes) attribute to specify the endpoint.